tests-configuration-infinispan/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/cache/container/scattered/cache/memory/OffHeapMemoryTest.java

```diff
@@ -72,12 +73,14 @@ public void initPage() {
     }
 
     @Test
+    @Ignore
     public void editCapacity() throws Exception {
         crudOperations.update(offHeapMemoryAddress(CACHE_CONTAINER, SCATTERED_CACHE), page.getOffHeapMemoryForm(), "capacity",
             Random.number());
     }
 
     @Test
+    @Ignore
```

Comment:
What is the reason for ignores in this class?

Reply:
UI model change, attribute (that is subject of testing) became deprecated